changes/2908-jrwalk.md

```diff
@@ -0,0 +1,4 @@
+Adds configuration logic to the `PydanticPluginConfig` constructor, enabling parsing of the configuration slots from `pyproject.toml` for `mypy>0.900`.
```

Comment:
No need to give the implementation details, just the enduser explanation
You can probably write something like
```
Make `pydantic-mypy` plugin compatible with `pyproject.toml`. see the [doc](...) for more information```

Reply:
changed in https://github.com/samuelcolvin/pydantic/pull/2908/commits/af411103cdc416f0e884ff2b3bee64c74c696506